content/doc/developer/publishing/releasing-cd.adoc

```diff
@@ -0,0 +1,154 @@
+---
+title: Setting up automated plugin release
+layout: developerguide
+references:
+- url: /jep/229
+  title: 'JEP-229: Continuous Delivery of Jenkins Components and Plugins'
+---
+
```

Comment:
```suggestion

WARNING: Continuous Delivery of Jenkins Components and Plugin (jep:229[]) is currently in preview.
See the referenced Jenkins enhancement proposal and references in it to see the current state.
Any feedback from early adopters will be appreciated.
```

Reply:
where is it marked as preview?

Reply:
```suggestion

NOTE: Continuous Delivery of Jenkins Components and Plugin (jep:229[]) is relatively new.
Any feedback from early adopters will be appreciated.
```

Reply:
Roadmap-wise, an initiative is in preview until the respective JEP is formally accepted

Reply:
https://github.com/jenkinsci/jep/tree/master/jep/14 does not state that as far as I can tell.

I don't think this discussion is for this pull request.

Nothing I can see anywhere states that this requires a preview disclaimer, I think the current disclaimer is appropriate.

Please take the bits about github actions long term to the mailing list as it doesn't belong in this PR

Reply:
FTR the public roadmap had a preview disclaimer until formal publishing and JEP acceptance approval by the Jenkins Governance Meeting and then by the BDFL delegate. See https://github.com/jenkins-infra/jenkins.io/pull/3527 for the pull request which removed the disclaimer

Reply:
That may be true but does not appear to be required?